<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="4" hostname="Dell-Namanmishar" name="com.employee.testCases.TC005_Delete_Employee_Record" tests="5" failures="0" timestamp="2021-04-08T19:51:51 IST" time="3.879" errors="1">
  <testcase name="checkstatusLine" time="0.000" classname="com.employee.testCases.TC005_Delete_Employee_Record">
    <skipped/>
  </testcase> <!-- checkstatusLine -->
  <system-out/>
  <testcase name="getAllEmployees" time="3.879" classname="com.employee.testCases.TC005_Delete_Employee_Record">
    <error type="io.restassured.path.json.exception.JsonPathException" message="Failed to parse the JSON document">
      <![CDATA[io.restassured.path.json.exception.JsonPathException: Failed to parse the JSON document
at io.restassured.path.json.JsonPath$ExceptionCatcher.invoke(JsonPath.java:1002)
at io.restassured.path.json.JsonPath$4.doParseWith(JsonPath.java:967)
at io.restassured.path.json.JsonPath$JsonParser.parseWith(JsonPath.java:1047)
at io.restassured.path.json.JsonPath.get(JsonPath.java:202)
at com.employee.testCases.TC005_Delete_Employee_Record.getAllEmployees(TC005_Delete_Employee_Record.java:28)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:62)
at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:385)
at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:321)
at org.testng.internal.TestMethodWorker.invokeBeforeClassMethods(TestMethodWorker.java:176)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:122)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
Caused by: java.lang.IllegalArgumentException: The JSON input text should neither be null nor empty.
at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
at sun.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source)
at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
at java.lang.reflect.Constructor.newInstance(Unknown Source)
at org.codehaus.groovy.reflection.CachedConstructor.invoke(CachedConstructor.java:72)
at org.codehaus.groovy.runtime.callsite.ConstructorSite$ConstructorSiteNoUnwrapNoCoerce.callConstructor(ConstructorSite.java:105)
at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCallConstructor(CallSiteArray.java:59)
at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callConstructor(AbstractCallSite.java:263)
at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callConstructor(AbstractCallSite.java:277)
at io.restassured.internal.path.json.ConfigurableJsonSlurper.parseText(ConfigurableJsonSlurper.groovy:80)
at io.restassured.path.json.JsonPath$4$1.method(JsonPath.java:965)
at io.restassured.path.json.JsonPath$ExceptionCatcher.invoke(JsonPath.java:1000)
... 17 more
]]>
    </error>
  </testcase> <!-- getAllEmployees -->
  <system-out/>
  <testcase name="checkStatusCode" time="0.000" classname="com.employee.testCases.TC005_Delete_Employee_Record">
    <skipped/>
  </testcase> <!-- checkStatusCode -->
  <system-out/>
  <testcase name="checkContentType" time="0.000" classname="com.employee.testCases.TC005_Delete_Employee_Record">
    <skipped/>
  </testcase> <!-- checkContentType -->
  <system-out/>
  <testcase name="checkResponseTime" time="0.000" classname="com.employee.testCases.TC005_Delete_Employee_Record">
    <skipped/>
  </testcase> <!-- checkResponseTime -->
  <system-out/>
</testsuite> <!-- com.employee.testCases.TC005_Delete_Employee_Record -->
